packages/e2e/nightwatch/tests/imtrFlow.js

```diff
@@ -0,0 +1,127 @@
+const { domain, selectors } = require("../config");
+
+const address = {
+  zipCode: "1055XD",
+  streetName: "Louise de Colignystraat",
+  houseNumber: 19,
+  houseNumberAddition: "C",
+  houseNumberFull: "19c",
+};
```

Comment:
maybe add a comment we should get this information from our mocking-package?

Reply:
I'm working on the implementation